_sass/app.scss

```diff
@@ -559,8 +559,19 @@ main {
 }
 
 // HOMEPAGE
+// Copied colour scheme from app.js
```

Comment:
Really pleased that this is now done using CSS. Hopefully some devices can then use hardware acceleration making it smoother than the JS one, at least on low powered devices.